src/SprykerEco/Yves/Payone/Theme/default/components/molecules/payone-klarna/payone-klarna.twig

```diff
@@ -0,0 +1,49 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'payone-klarna',
+    tag: 'payone-klarna',
+} %}
+
+{% define data = {
+    form: required,
+    billingAddressData: data.billingAddressData | default({}),
+    customerData: data.customerData | default({}),
```

Comment:
Please remove this property as useless and use the `'date-of-birth'` attribute directly.